source/location.go

```diff
@@ -907,3 +912,18 @@ func MessageDependencyArgumentInlineLocation(fileName, msgName string, idx, argI
 		},
 	}
 }
+
+// MessageValidationLocation creates location for messages[*].validations[*] in grpc.federation.message.
```

Comment:
`s/messages[*].validations[*]/validations[*]/` ? ( `validations` is top level field of `grpc.federation.message` option )